bundlers/webpack-browser-custom-output/index.test.js

```diff
@@ -0,0 +1,7 @@
+const { Role } = require('./dist/prismaTest')
+
+test('correctly generates browser-compatible bundle for prisma at custom path', () => {
+  expect(Role).toBeDefined()
+  expect(Role.USER).toBeDefined()
+  expect(Role.ADMIN).toBeDefined()
```

Comment:
Is this enough testing to make sure Prisma Client actually works? Should it maybe try to do a query and confirm the data returned etc?

Reply:
Are the other tests in the `bundler` folder similar? Some of the CI pipeline certainly expects more, as also seen via the `schema.prisma` file here and so on and the tests failing because of missing engines etc.

Reply:
No, other bundler target node.js, where client is available. The whole purpose of this test is to test that bundling for browser also works (prisma/prisma#12296). Do you think it'll make sense create a separate category for those kinds of tests (`bundler-browser`?) to make that distinction clearer?

Reply:
@millsp will know more if we already have something like this in another test. Generally the whole repo was set up to test if Prisma Client works in certain situations (databases, deployments and so on).

Reply:
@millsp was the one who sent me to this repo.

Reply:
The best way to check if it's the browser build running is to do `new PrismaClient` and check that it throws an error saying `PrismaClient is unable to be run in the browser`.

Reply:
@janpio @millsp Updated the test to test for that exception